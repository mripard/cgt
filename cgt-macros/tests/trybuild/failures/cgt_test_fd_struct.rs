#[cgt_macros::cgt_test_with_fd]
struct Dummy;