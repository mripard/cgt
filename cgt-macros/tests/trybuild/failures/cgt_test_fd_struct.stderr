error: expected `fn`
 --> tests/trybuild/failures/cgt_test_fd_struct.rs:2:1
  |
2 | struct Dummy;
  | ^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/trybuild/failures/cgt_test_fd_struct.rs:2:14
  |
2 | struct Dummy;
  |              ^ consider adding a `main` function to `$DIR/tests/trybuild/failures/cgt_test_fd_struct.rs`
